File: 프로그래머스/lv0/120813. 짝수는 싫어요/짝수는 싫어요.py
def solution(n):
    answer = []
    for i in range(n+1) :
        if i%2 != 0 :
            answer.append(i)
    return answer

# 다른 사람의 풀이
# def solution(n):
#     return [i for i in range(1, n+1, 2)]

File: 프로그래머스/0/120813. 짝수는 싫어요/짝수는 싫어요.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int* solution(int n) {
    // return 값은 malloc 등 동적 할당을 사용해주세요. 할당 길이는 상황에 맞게 변경해주세요.
    int a = 0;
    int* answer = (int*)malloc(((n/2)+1) * sizeof(int));
    for(int i=1; i<=n; i+=2){
        answer[a] = i;
        a++;
    }
    return answer;
}